src/main/resources/liquibase/crop_changelog/13_3_0.xml

```diff
@@ -25,4 +25,117 @@
 		</sql>
 	</changeSet>
 
+	<changeSet author="vanina" id="v13.3.0-2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="ims_lot" columnName="lot_uuid"/>
+			</not>
+		</preConditions>
+		<comment>
+			Add column lot_uuid to ims_lot table
+		</comment>
+		<addColumn tableName="ims_lot">
+			<column name="lot_uuid" type="VARCHAR(36)">
+				<constraints nullable="true" unique="true"/>
+			</column>
+		</addColumn>
+	</changeSet>
+
+	<changeSet author="vanina" id="v13.3.0-3">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT
+				COUNT(*)
+				FROM
+				information_schema.routines
+				WHERE
+				routine_schema = DATABASE()
+				AND specific_name = 'upgrade_lot_id';
+			</sqlCheck>
+		</preConditions>
+		<sql dbms="mysql" splitStatements="false">
+			DROP PROCEDURE upgrade_lot_id;
+		</sql>
+	</changeSet>
+
+	<changeSet author="vanina" id="v13.3.0-4">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT
+				COUNT(*)
+				FROM
+				information_schema.routines
+				WHERE
+				routine_schema = DATABASE()
+				AND specific_name = 'upgrade_lot_id';
+			</sqlCheck>
+		</preConditions>
+		<sql dbms="mysql" splitStatements="false">
+			CREATE PROCEDURE upgrade_lot_id()
+			BEGIN
+
+			DECLARE useUuid INT DEFAULT 1;
+
+			SELECT COUNT(*)
```

Comment:
@vmaletta this value must be queried per crop given that it is stored per crop in workbench_crop table

Reply:
Done